smoketests/tests/servers.py

```diff
@@ -23,3 +23,15 @@ def test_servers(self):
 
         out = self.spacetime("server", "fingerprint", "-s", "localhost")
         self.assertIn("Fingerprint is unchanged for server localhost", out)
+
+    def test_edit_server(self):
```

Comment:
This test was originally part of https://github.com/clockworklabs/SpacetimeDB/pull/1845/files#diff-a953574fc278eedbeb3eb9090a80a7fcc97d5ea94a1e5612b25d54d877c43a4a, but I moved it here to test the current behavior.